sqlstats/sqlstats.go

```diff
@@ -70,14 +83,27 @@ func (t *Tracer) done(s *connStats) (why string, readOnly bool) {
 	return why, readOnly
 }
 
-type queryStats struct {
-	query string
+// QueryStats is a collection of stats for a given Query.
+type QueryStats struct {
+	Query string
 
 	// When inside the queries map all fields must be accessed as atomics.
-	count    int64
-	errors   int64
-	duration int64 // time.Duration
-	mean     int64
+
+	// Count represents the number of times this query has been
+	// executed during a trace.
```

Comment:
The tracer is passed to sqlite.Connector and used for the life of the database/sql.DB - "during a trace" feels a bit short-lived in that context.